Assets/HoloToolkit/Input/Scripts/InputSources/BaseSingletonInputSource.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I agree. Once managers become input sources or focus sources they won't need to be singletons. This was just a hack to ease the transition.